managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorImpl.java

```diff
@@ -1850,6 +1831,10 @@ public void operationFailed(ManagedLedgerException exception) {
         });
     }
 
+    protected void persistPosition(MarkDeleteEntry mdEntry, VoidCallback callback) {
```

Comment:
A minor question, why we need this wrapper?

Reply:
`RemoteManagedCursorImpl` overrided this method and use CursorClient to persist this 'mdEntry' to topic write owner.